app/src/main/java/swati4star/createpdf/fragment/PdfToImageFragment.java

```diff
@@ -1,15 +1,17 @@
 package swati4star.createpdf.fragment;
 
 
+import static android.app.Activity.RESULT_OK;
+import static swati4star.createpdf.util.Constants.BUNDLE_DATA;
+import static swati4star.createpdf.util.Constants.PDF_TO_IMAGES;
+import static swati4star.createpdf.util.Constants.REQUEST_CODE_FOR_WRITE_PERMISSION;
+import static swati4star.createpdf.util.Constants.WRITE_PERMISSIONS;
+
```

Comment:
Revert these unnecessary changes! (Any beautification work should be done in separate commit may or may not in single pull request)